docs/external-node.md

```diff
@@ -334,6 +338,70 @@ please refer to the [getting-started guide](getting-started.md#open-vswitch).
       sudo systemctl start antrea-agent
       ```
 
+##### Container Installation
+
+1. The Docker image can be built from source code or can be downloaded from the antrea repository.
+
+   1. From Source
+   
+      Build `antrea-ubuntu` docker image in the root of the antrea code tree.
+
+      ```bash
+      make
+      ```
+
+      Rename that image to `projects.registry.vmware.com/antrea/antrea-ubuntu:latest`
+
+      ```bash
+      docker image tag <image-id> projects.registry.vmware.com/antrea/antrea-ubuntu:latest
+      ```
+
+      Copy the `projects.registry.vmware.com/antrea/antrea-ubuntu:latest` image to install VM. Please follow the below steps.
+
+      ```bash
+      # Save it in a tar file
+      docker save -o <tar file path in source host machine> projects.registry.vmware.com/antrea/antrea-ubuntu
+ 
+      # Copy this tar file using simple tools such as cp, rsync, scp, etc, or any other method that you prefer.
```

Comment:
I feel no need to give examples of commands. Maybe remove "using simple tools...". Just say "Copy this tar file to the target VM".

Reply:
Done.